integration_tests/pages/barcode/chooseBarcodeOption.ts

```diff
@@ -1,6 +1,8 @@
+/* eslint-disable import/no-cycle */
```

Comment:
This is a major downside - each page knows about both its previous and next pages resulting in a dependency cycle. On the plus side it only uses a static member from its previous page but this could encourage use of non-static members.

I guess the alternative is to have a God class that knows about all pages but that could get messy.

Any better ideas?